### PR TITLE
Rename votes to stakes

### LINKED ISSUE #4628: Rename Votes to Stakes
### Description

Votes changed to the staker so we need to follow same terminology in our app

### Acceptance Criteria
- Rename component 
- Ensure tooltip, titles and all other texts are updated, cross check the Figma for updated texts
- Adopt variable name
- Adopt UI text
- Adopt unit test

### Additional Information
- voteDelegate  -> stakeValidator
- Vote  -> stake
- Voter  -> staker
- self-vote  -> self-stake 
- unvote  ->unstake

### PR DESCRIPTION
**Pull request by oskarleonard**
### What was the problem?

This PR resolves #4628

### How was it solved?

- Renamed folders
- Renamed components, hooks, etc
- Updated imports

### How was it tested?

Unit tests

### CODE REVIEW

File: src/modules/account/components/AccountCreationTips/index.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ const AccountCreationTips = () => {
             <span className={styles.number}>02</span>
             <div className={styles.tip}>
               <h5>{t('Participate in blockchain governance')}</h5>
-              <span>{t('Vote for your trusted validator or become a validator to secure the blockchain.')}</span>
+              <span>{t('Stake for your trusted validator or become a validator to secure the blockchain.')}</span>
```
Comment (reyraa) [inline src/modules/account/components/AccountCreationTips/index.js:61]: I doubt this is grammatically correct.

File: src/modules/common/__fixtures__/mockNewsFeed.js [hunk 2]
```diff
@@ -133,7 +133,7 @@ export const mockNewsFeed = {
     url: 'https://lisk.com/blog/research/where-store-lsk-lisk-desktop-and-lisk-mobile-wallets',
   }, {
     author: 'Tony Morinello',
-    content: 'Choosing which validators to vote for is a crucial component of Lisk\'s Validatord\nProof of Stake (DPoS) consensus',
+    content: 'Choosing which validators to stake for is a crucial component of Lisk\'s Validatord\nProof of Stake (DPoS) consensus',
```
Comment (reyraa) [inline src/modules/common/__fixtures__/mockNewsFeed.js]: dangling preposition is discouraged.  We can change this to
> A crucial component of Lisk's Proof of Stake (PoS) consensus is choosing reliable validators for which you stake tokens.

File: src/modules/common/components/bars/topBar/stakeQueueToggle.js [hunk 3]
```diff
@@ -15,20 +15,20 @@ const VoteQueueToggle = ({
...
+        component="stakingQueue"
+        className={`${styles.toggle} staking-queue-toggle ${
           disabled && `${styles.disabled} disabled`
         }`}
       >
-        <Icon name="votingQueueInactive" />
+        <Icon name="stakingQueueInactive" />
         {noOfVotes !== 0 && (
-          <span className={styles.votingQueueVoteCount}>{noOfVotes}</span>
+          <span className={styles.stakingQueueVoteCount}>{noOfVotes}</span>
```
Comment (reyraa) [inline src/modules/common/components/bars/topBar/stakeQueueToggle.js]: This seems to be a mix of vote and stake

Comment (reyraa) [reply src/modules/common/components/bars/topBar/stakeQueueToggle.js]: noOfVote -> statekCount
votingQueueVoteCount -> queueLength

File: src/modules/pos/validator/components/EditStake/index.js [hunk 4]
```diff
@@ -120,15 +120,15 @@ const EditVote = ({ history, voteEdited, network, voting, votesRetrieved }) => {
   }, [token, auth, network, voting]);
 
   useEffect(() => {
-    votesRetrieved();
+    stakesRetrieved();
```
Comment (reyraa) [inline src/modules/pos/validator/components/EditStake/index.js:123]: Also, I think we should go with `stakeList` instead of `stakingQueue`.

File: src/modules/pos/validator/components/EditStake/index.js [hunk 5]
```diff
@@ -120,15 +120,15 @@ const EditVote = ({ history, voteEdited, network, voting, votesRetrieved }) => {
...
   }, []);
 
   const handleConfirm = () => {
     if (!isForm) {
-      removeThenAppendSearchParamsToUrl(history, { modal: 'votingQueue' }, ['modal']);
+      removeThenAppendSearchParamsToUrl(history, { modal: 'stakingQueue' }, ['modal']);
```
Comment (reyraa) [inline src/modules/pos/validator/components/EditStake/index.js:128]: Also, I think we should go with `stakeList` instead of `stakingQueue`.

File: src/modules/pos/validator/components/EditStake/index.test.js [hunk 6]
```diff
@@ -8,12 +8,12 @@ import mockSavedAccounts from '@tests/fixtures/accounts';
 import { mockBlocks } from '@block/__fixtures__';
 import { useAuth } from '@auth/hooks/queries';
 import { useTokensBalance } from '@token/fungible/hooks/queries';
-import { mockValidators, mockSentVotes } from '@pos/validator/__fixtures__';
+import { mockValidators, mockSentStakes } from '@pos/validator/__fixtures__';
```
Comment (reyraa) [inline src/modules/pos/validator/components/EditStake/index.test.js:11]: you send votes to validators and receive votes from the community. 
Is that grammatically valid for stakes too?

File: src/modules/pos/validator/components/StakeStatus/statusMessages.js [hunk 7]
```diff
@@ -28,10 +28,10 @@ const getSuccessMessage = (t, locked, unlockable, selfUnvote = { confirmed: 0 })
               <span>{t('will be available to unlock in {{unlockTime}}h.', { unlockTime })}</span>
             </>
           ) : null}
-        {selfUnvoteUnlockable > 0
+        {selfUnStakeUnlockable > 0
```
Comment (reyraa) [inline src/modules/pos/validator/components/StakeStatus/statusMessages.js]: `Un` is a prefix not a word. I think it should be `Unstake`

File: src/modules/pos/validator/components/StakeStatus/statusMessages.js [hunk 8]
```diff
@@ -64,12 +64,12 @@ const getSuccessMessage = (t, locked, unlockable, selfUnvote = { confirmed: 0 })
   return '';
 };
 
-const voteStatusMessages = (t, statusInfo) => ({
+const stakeStatusMessages = (t, statusInfo) => ({
   ...statusMessages(t),
   [txStatusTypes.broadcastSuccess]: {
-    title: t('Votes are submitted'),
+    title: t('Stakes are submitted'),
```
Comment (reyraa) [inline src/modules/pos/validator/components/StakeStatus/statusMessages.js]: Should it be `Tokens are staked` ? You submit your vote, but do we say the same way about stakes?

File: src/modules/pos/validator/components/StakeSuccessfulModal/StakeSuccessfulModal.js [hunk 9]
```diff
@@ -18,10 +18,10 @@ function VoteSuccessfulModal({ history, statusMessage }) {
 
   return (
     <Box className={styles.successContainer}>
-      <BoxHeader>{t('Voting confirmation')}</BoxHeader>
+      <BoxHeader>{t('Staking confirmation')}</BoxHeader>
       <BoxContent>
         <Illustration className={styles.illustartion} name="votingSuccess" />
-        <h4>{t('Vote(s) has been submitted')}</h4>
+        <h4>{t('Stake(s) has been submitted')}</h4>
```
Comment (reyraa) [inline src/modules/pos/validator/components/StakeSuccessfulModal/StakeSuccessfulModal.js]: Same here. We might be able to change this to
> Tokens are now staked.

File: src/locales/en/common.json [hunk 10]
```diff
@@ -416,7 +416,7 @@
   "Please enter a valid generator key value": "Please enter a valid generator key value",
   "Please enter a valid proof of possession value": "Please enter a valid proof of possession value",
   "Please enter a valid public key for each member.": "Please enter a valid public key for each member.",
-  "Please enter vote amounts for the validators you wish to vote for": "Please enter vote amounts for the validators you wish to vote for",
+  "Please enter stake amounts for the validators you wish to stake for": "Please enter stake amounts for the validators you wish to stake for",
```
Comment (chris529) [inline src/locales/en/common.json]:
```suggestion
  "Please enter the stake amounts for the validators you wish to stake for": "Please enter the stake amounts for the validators you wish to stake for",
```

File: src/locales/en/common.json [hunk 11]
```diff
@@ -565,12 +582,12 @@
   "The message signature has been canceled on your {{model}}": "The message signature has been canceled on your {{model}}",
   "The minimum required balance for this action is {{minRequiredBalance}} {{token}}": "The minimum required balance for this action is {{minRequiredBalance}} {{token}}",
   "The number of transactions submitted over the selected time period.": "The number of transactions submitted over the selected time period.",
-  "The provided amount is higher than available voting balance.": "The provided amount is higher than available voting balance.",
+  "The provided amount is higher than available staking balance.": "The provided amount is higher than available staking balance.",
```
Comment (chris529) [inline src/locales/en/common.json]:
```suggestion
  "The provided amount is higher than the available staking balance.": "The provided amount is higher than the available staking balance.",
```

File: src/locales/en/common.json [hunk 12]
```diff
@@ -586,11 +603,11 @@
...
   "There are no transactions for this chain.": "There are no transactions for this chain.",
   "There is no connection to the {{model}}. Please check the cables if it happened by accident.": "There is no connection to the {{model}}. Please check the cables if it happened by accident.",
+  "This account does not have any staker for the given address.": "This account does not have any staker for the given address.",
+  "This account does not have any stakers.": "This account does not have any stakers.",
   "This account does not have any transactions.": "This account does not have any transactions.",
-  "This account does not have any voter for the given address.": "This account does not have any voter for the given address.",
-  "This account does not have any voters.": "This account does not have any voters.",
-  "This account doesn’t have any votes matching searched username.": "This account doesn’t have any votes matching searched username.",
-  "This account doesn’t have any votes.": "This account doesn’t have any votes.",
+  "This account doesn’t have any stakes matching searched username.": "This account doesn’t have any stakes matching searched username.",
```
Comment (chris529) [inline src/locales/en/common.json]:
```suggestion
  "This account doesn’t have any stakes matching the searched username.": "This account doesn’t have any stakes matching the searched username.",
```

File: src/modules/account/components/AccountCreationTips/index.js (hunk 1 above)
Comment (chris529) [reply src/modules/account/components/AccountCreationTips/index.js:61]: imho it is fine. However, if you wish to change it another option we could use is as follows:
"You may stake for your trusted validator or become a validator to secure the blockchain"

File: src/modules/common/__fixtures__/mockNewsFeed.js (hunk 2 above)
Comment (chris529) [reply src/modules/common/__fixtures__/mockNewsFeed.js]: imho, I would suggest we reword this to the following:
"A crucial component of Lisk's Proof of Stake (PoS) consensus is choosing reliable validators with whom you can stake your tokens."

File: src/modules/common/components/onboarding/demo.js [hunk 13]
```diff
@@ -11,7 +11,7 @@ const getOnboardingSlides = () => ([{
   illustration: 'welcomeLiskValidators',
 }, {
   title: ('Your voice matters'),
-  content: ('In this section of Lisk you can vote for up to 101 validators to run Lisk’s blockchain network and by doing so have a real impact on the Lisk ecosystem.'),
+  content: ('In this section of Lisk you can stake for up to 101 validators to run Lisk’s blockchain network and by doing so have a real impact on the Lisk ecosystem.'),
```
Comment (chris529) [inline src/modules/common/components/onboarding/demo.js]:
```suggestion
  content: ('In this section of Lisk you can stake for up to 101 validators to run Lisk’s blockchain network, and by doing so have a real impact on the Lisk ecosystem.'),
```

File: src/modules/pos/validator/components/EditStake/index.test.js (hunk 6 above)
Comment (chris529) [reply src/modules/pos/validator/components/EditStake/index.test.js:11]: imho I would say not necessarily, therefore it would be helpful to see the exact context of the sentence you wish to use first, then we could determine this correctly.

File: src/modules/pos/validator/components/EditStake/index.test.js [hunk 14]
```diff
@@ -88,21 +88,21 @@ describe('EditVote', () => {
     ).toBeTruthy();
     expect(
       screen.getByText(
-        'Insert a vote amount for this validator. Your new vote will be added to the voting queue.'
+        'Insert a stake amount for this validator. Your new stake will be added to the staking queue.'
```
Comment (chris529) [inline src/modules/pos/validator/components/EditStake/index.test.js]:
```suggestion
        'Insert an amount you wish to stake for this validator. Your new stake will then be added to the staking queue.'
```

File: src/modules/pos/validator/components/StakeForm/StakeForm.js [hunk 15]
```diff
@@ -3,45 +3,46 @@ import React, { useMemo, useState } from 'react';
...
+import StakeRow from './StakeRow';
 import EmptyState from './EmptyState';
 import header from './tableHeader';
-import styles from './voteForm.css';
+import styles from './stakeForm.css';
 
 /**
- * Determines the number of votes that have been
+ * Determines the number of stakes that have been
  * added, removed or edited.
```
Comment (chris529) [inline src/modules/pos/validator/components/StakeForm/StakeForm.js]:
```suggestion
 * added, removed, or edited.
```

File: src/modules/pos/validator/components/StakeForm/StakeForm.js [hunk 16]
```diff
@@ -58,47 +59,47 @@ const getVoteStats = (votes, account) => {
...
-const validateVotes = (votes, balance, fee, resultingNumOfVotes, t, dposToken) => {
+const validateStakes = (votes, balance, fee, resultingNumOfStakes, t, dposToken) => {
   const messages = [];
   const areVotesInValid = Object.values(votes).some(
     (vote) => vote.unconfirmed === '' || vote.unconfirmed === undefined
   );
 
   if (areVotesInValid) {
-    messages.push(t('Please enter vote amounts for the validators you wish to vote for'));
+    messages.push(t('Please enter stake amounts for the validators you wish to stake for'));
```
Comment (chris529) [inline src/modules/pos/validator/components/StakeForm/StakeForm.js]:
```suggestion
    messages.push(t('Please enter the stake amounts for the validators you wish to stake for'));
```

File: src/modules/pos/validator/components/StakeStatus/statusMessages.js (hunk 7 above)
Comment (chris529) [reply src/modules/pos/validator/components/StakeStatus/statusMessages.js]: Agreed 👍

File: src/modules/pos/validator/components/StakeStatus/statusMessages.js (hunk 8 above)
Comment (chris529) [reply src/modules/pos/validator/components/StakeStatus/statusMessages.js]: Yes, you are correct Ali. It should be "Tokens are staked"

File: src/modules/pos/validator/components/StakeSuccessfulModal/StakeSuccessfulModal.js (hunk 9 above)
Comment (chris529) [reply src/modules/pos/validator/components/StakeSuccessfulModal/StakeSuccessfulModal.js]: Agreed, imho this should be changed to the following:
"Tokens are now staked"

File: src/modules/pos/validator/store/actions/staking.js [hunk 17]
```diff
@@ -27,35 +27,35 @@ export const votesCleared = () => ({
...
  */
-export const votesConfirmed = () => ({
-  type: actionTypes.votesConfirmed,
+export const stakesConfirmed = () => ({
+  type: actionTypes.stakesConfirmed,
 });
 
 /**
- * To be dispatched when a vote is to be removed form the voting queue
+ * To be dispatched when a vote is to be removed form the staking queue
```
Comment (chris529) [inline src/modules/pos/validator/store/actions/staking.js]:
```suggestion
 * To be dispatched when a vote is to be removed from the staking queue
```

File: src/modules/wallet/components/stakes/stakes.js [hunk 18]
```diff
@@ -7,18 +7,18 @@ import BoxHeader from '@theme/box/header';
...
+import styles from './stakes.css';
+import StakeRow from './stakeRow';
+import header from './stakesTableHeader';
 
 const getMessages = (t) => ({
-  all: t('This account doesn’t have any votes.'),
+  all: t('This account doesn’t have any stakes.'),
   filtered: t(
-    'This account doesn’t have any votes matching searched username.',
+    'This account doesn’t have any stakes matching searched username.',
```
Comment (chris529) [inline src/modules/wallet/components/stakes/stakes.js]:
```suggestion
    'This account doesn’t have any stakes matching the searched username.',
```

File: src/utils/validators.js [hunk 19]
```diff
@@ -104,16 +104,16 @@ export const validateAmountFormat = ({
...
+      message: i18n.t('You can only stake in multiplies of 10 LSK.'),
       fn: () => value % 10 !== 0,
     },
     INSUFFICIENT_FUNDS: {
       message: i18n.t('Provided amount is higher than your current balance.'),
       fn: () => funds < toRawLsk(numeral(value).value()),
     },
     INSUFFICIENT_VOTE_FUNDS: {
-      message: i18n.t('The provided amount is higher than available voting balance.'),
+      message: i18n.t('The provided amount is higher than available staking balance.'),
```
Comment (chris529) [inline src/utils/validators.js]:
```suggestion
      message: i18n.t('The provided amount is higher than your available staking balance.'),
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

File: src/modules/common/components/bars/topBar/stakeQueueToggle.js (hunk 3 above)
Comment (oskarleonard) [reply src/modules/common/components/bars/topBar/stakeQueueToggle.js]: https://github.com/LiskHQ/lisk-desktop/pull/4670/commits/59498601dafc0cc81623c7a4c86fc5f9885652be

**Review by reyraa — DISMISSED** [review]
LGTM

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Review by reyraa — APPROVED** [review]
